core/libs/shared/src/core_tree.rs

```diff
@@ -8,6 +8,7 @@ use hmdb::transaction::TransactionTable;
 use std::collections::HashSet;
 use tracing::error;
 use uuid::Uuid;
+extern crate tracing;
```

Comment:
Shouldn't this appear once in the lib.rs file instead of each file that needs it? Also, does this file even need it?

Reply:
not even needed